ikasaneip/wiretap/src/test/java/org/ikasan/wiretap/dao/HibernateWiretapDaoTest.java

```diff
@@ -73,75 +68,89 @@
 	/** Object being tested */
 	@Resource private WiretapDao wiretapDao;
 
-	/**
-	 * Before each test case, inject a mock {@link HibernateTemplate} to dao implementation
-	 * being tested
-	 */
-	@Before
-	public void setup()
-	{
-
-		for(int i=0; i< 10000; i++)
-		{
-			WiretapFlowEvent event = new WiretapFlowEvent("moduleName", "flowName", "componentName",
-					"eventId", "relatedEventId", System.currentTimeMillis() ,"event", System.currentTimeMillis() - 1000000000);
-
-			this.wiretapDao.save(event);
-		}
-
-	}
+//	/**
+//	 * Before each test case, inject a mock {@link HibernateTemplate} to hibernate implementation
+//	 * being tested
+//	 */
+//	@Before
+//	public void setup()
+//	{
+//
+//		for(int i=0; i< 10000; i++)
+//		{
+//			WiretapFlowEvent event = new WiretapFlowEvent("moduleName", "flowName", "componentName",
+//					"eventId", "relatedEventId", System.currentTimeMillis() ,"event", System.currentTimeMillis() - 1000000000);
+//
+//			this.wiretapDao.save(event);
+//		}
+//
+//	}
 
 	@Test
 	@DirtiesContext
 	public void test_get_harvestableRecords()
 	{
```

Comment:
given test has no assertions. only system outs.  Can we revert to previous version which was asserting ?

Reply:
As I have stated many times @andrzej-majewski I simply checked in a bunch of work mid development as I was stepping away from the computer and did not want to risk losing work. This was clearly set up to point to our Sybase database and the unit test used to bootstrap my ability to debug hibernate to work out how the implementation worked against Sybase and in particular how I could extend the dialect. Moreover I did not create a pull request nor ask you to review the work. So these comments, albeit correct and an oversight on my behalf, are somewhat unnecessary, as I never claimed to be dev complete, nor requested a review.